src/lib/PreviewUI.js

```diff
@@ -311,6 +312,24 @@ class PreviewUI {
         this.notification.hide();
     }
 
+    /**
+     * Replaces the currently active header with a specified header
+     *
+     * @public
+     * @param {string} replacementHeader - Class name of new header
+     * @return {void}
+     */
+    replaceHeader(replacementHeader) {
+        // First hide all possible headers
+        this.container.querySelectorAll(`.${CLASS_HIDDEN}`).forEach((element) => {
```

Comment:
Do you mean to add `CLASS_HIDDEN` to all elements that have `CLASS_HIDDEN`?
Shouldn't this just be selecting the header element and hiding it, if it exists?